### DISCUSSION TITLE
How could I change the profile photo (because it's not loading) and change the Edge modules for Firefox, open the files with Thunar

### DISCUSSION THREAD

**Discussion by kuberpunk666** [Jas-SinghFSU/HyprPanel#6] [Q&A] [answered]
![swappy-20240726_134458](https://github.com/user-attachments/assets/561d2c2a-a274-47ff-aff3-c85ee35faff4)

Beautiful rice.  It was incredible! One of the most beautiful I've ever seen :)

**Comment by Jas-SinghFSU** [comment, marked as answer]
You should be able to change all of that in the settings menu. If you click the Gear icon in the dashboard, the go to Configuration > Dashboard Menu, all options to change everything in the dashboard are there.
